drivers/azure/azure.go

```diff
@@ -150,6 +170,11 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
 			Value:  defaultAzureImage,
 		},
 		mcnflag.StringFlag{
+			Name:  flAzureOS,
+			Usage: "OS for the Azure VM (Windows|Linux)",
```

Comment:
I think this field is case-sensitive, so these docs are going to be wrong. Let's just say `(allowed values: "windows", "linux")`